shared/src/models/utils/position/map-position.ts

```diff
@@ -0,0 +1,31 @@
+import { Type } from 'class-transformer';
+import { ValidateNested } from 'class-validator';
+import { IsValue } from '../../../utils/validators';
+import { getCreate } from '../get-create';
+import { MapCoordinates } from './map-coordinates';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { isOnMap, isNotOnMap, coordinatesOf } from './position-helpers';
```

Comment:
To have them highlighted in the @link descriptions in comments.

Reply:
Could you add a comment in the line above the `eslint-disable`? I think this would help us understanding why these exports are necessary in the future.